packages/http-client-python/emitter/src/lib.ts

```diff
@@ -49,7 +49,14 @@ const EmitterOptionsSchema: JSONSchemaType<PythonEmitterOptions> = {
 
 const libDef = {
   name: "@typespec/http-client-python",
-  diagnostics: {},
+  diagnostics: {
+    "no-valid-client": {
+      severity: "warning",
```

Comment:
just one concern. if python does not support model only generation, shall we report error here?